src/adapters/EnzymeAdapter.js

```diff
@@ -0,0 +1,24 @@
+function unimplementedError(methodName, classname) {
+  return new Error(
+    `${methodName} is a required method of ${classname}, but was not implemented.`,
+  );
+}
+
+class EnzymeAdapter {
```

Comment:
i don't see anywhere we're checking `instanceof EnzymeAdapter` - it seems like we'd want to enforce people use this interface.